tests/cli-app-permission-helper.test.ts

```
@@ -3,6 +3,7 @@ import { beforeEach, describe, expect, it, vi } from 'vitest'
 const hasCliPermissionMock = vi.hoisted(() => vi.fn())
 
 vi.mock('../cli/src/utils', () => ({
+  appAddHintMessage: (appId: string) => `App ${appId} does not exist`,
```

Biome (2.5.6)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)